spec/betagouvbot/webhook_spec.rb

```diff
@@ -3,10 +3,11 @@
 
 RSpec.describe BetaGouvBot::Webhook do
   describe '/compte' do
-    let(:callback)  { 'https://bob.coop' }
-    let(:user_name) { 'bob' }
-    let(:text)      { "#{user_name} #{user_name}@email.coop password" }
-    let(:params)    { { response_url: callback, user_name: user_name, text: text } }
+    let(:callback)     { 'https://bob.coop' }
+    let(:user_name)    { 'bob' }
+    let(:text)         { "#{user_name} #{user_name}@email.coop password" }
+    let(:empty_params) { { response_url: callback, user_name: user_name } }
+    let(:valid_params) { empty_params.merge(text: text) }
```

Comment:
The Slack API doesn't respect REST standards. We don't have to like it, and we're under no obligation to use Slack to interact with BGB, but it's unlikely we'll get them to change their implementation.